tests/python/pants_test/logging/test_workunit_label.py

```diff
@@ -15,18 +15,19 @@ class WorkUnitLabelTest(PantsRunIntegrationTest):
 
   def test_workunit_no_label_ignore(self):
     pants_run = self.run_pants([
-      '--pythonpath={}'.format(os.path.join(os.getcwd(), 'tests', 'python')),
-      '--backend-packages={}'.format('pants_test.logging.data'),
+      '--pythonpath=%(buildroot)s/tests/python',
```

Comment:
Is this supposed to be an f-string?

Reply:
Originally, no. I had gotten it from:

https://github.com/pantsbuild/pants/blob/af6bfbbb232c0530c0c9e927b40d3d1bdc5381e5/tests/python/pants_test/task/test_goal_options_mixin_integration.py#L27-L31

Realized that that string interpolation wouldn't work for CLI args, though.. Will revert back to the original.